src/shared/utils/numbers/shortify.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Makes large numbers shorter. Examples:
+ * - 999 -> 999
+ * - 1000 -> 1K
+ * - 1000000 -> 1M
+ * - 1000000000 -> 1B
+ */
+export const shortify = (value: number): string => {
```

Comment:
There's a package `friendly-numbers`. Perhaps it's more wise to use that.

Reply:
But it does just the same job, only the 'T' case (trillions) is not covered by me. Why would we need a library for that?